compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/DataClassMembersGenerator.kt

```diff
@@ -254,13 +256,17 @@ class DataClassMembersGenerator(
                     irLetS(
```

Comment:
I think with your comments to irLetS this might not be needed. It looks like irLetS will just return the symbol for the IrGet and therefore I don't think it is needed for this?

Reply:
irGet here produces a call to a getter, not a read of a variable, so irLetS is necessary for exactly-once evaluation.

Reply:
Hah! Thanks Denis, yes, I totally misunderstood that. :)